.evergreen/run-static-checks.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o xtrace   # Write all commands first to stderr
+set -o errexit  # Exit the script with error if any of the commands fail
+
+export JAVA_HOME=/opt/java/jdk11
+
+############################################
+#            Main Program                  #
+############################################
+
+echo "Compiling and running checks"
+
+# We always compile with the latest version of java
+./gradlew -version
+./gradlew --info -x jvmTest -x nativeTest -x jsNodeTest clean check
```

Comment:
Should we add whatever the equivalent is of `jar testClasses docs`, as in the java build?

Reply:
check compiles all classes and runs spotless checks. There are no docs yet as dokka has not been setup.

Reply:
I can't remember then why we had to include testClasses in the Java build.

Reply:
Added KBSON-8 to cover api docs